src/tools/genpolicy/src/utils.rs

```diff
@@ -64,6 +64,17 @@ struct CommandLineOptions {
         help = "Ignore unsupported input Kubernetes YAML fields. This is not recommeded unless you understand exactly how genpolicy works!"
     )]
     silent_unsupported_fields: bool,
+
+    #[clap(
```

Comment:
Please add information about this new parameter in src/tools/genpolicy/genpolicy-advanced-command-line-parameters.md

Reply:
Addressed in https://github.com/kata-containers/kata-containers/pull/9185/files#diff-7d4751ed1162adb9a4cebb5492754adbce924ab9be9a698004416f3f670f1e27R60